draft-ietf-quic-ack-frequency.md

```diff
@@ -479,8 +479,14 @@ new ACK_FREQUENCY frame immediately upon confirmation of connection migration.
 
 
 # Security Considerations
+
 TBD.
 
+AN Improperly configured or malicious sender could cause standards-compliant receivers to 
+acknowledge more frequently than their available resources permit, thus causing a 
+Denial Of Service (DOS) attack. However, A sender has control of many aspects of 
+the receiver behaviour and therefore there is no additional threat from a well-configured sender. 
```

Comment:
s/there is no additional threat from a well-configured sender/for a well-configured sender this causes no additional threat/